src/libs/browser-storage.js

```diff
@@ -51,7 +51,7 @@ export const listenStatic = (storage, key, fn) => {
 export const getDynamic = (storage, key) => {
   const storageKey = `dynamic-storage/${key}`
   const data = maybeParseJSON(storage.getItem(storageKey))
-  return data && data.value
+  return data?.value
```

Comment:
So, since `maybeParseJson` will default to `undefined`, this shouldn't be an issue; either the item is unset, in which case it's always returned `undefined`, or it's an object; there aren't any other falsy values possible here.